ext/opentelemetry-ext-opentracing-shim/src/opentelemetry/ext/opentracing_shim/__init__.py

```diff
@@ -82,8 +82,16 @@
 
 import logging
 
-import opentracing
 from deprecated import deprecated
+from opentracing import (  # pylint: disable=no-name-in-module
```

Comment:
non blocking, but i'd prefer we keep the opentracing import and namespacing as it was before, especially if it has the benefit of not needing to add the disable statement above

Reply:
All right! will fix this :+1:

Reply:
In fact, after adding the changes the pylint disable statement would be necessary in many more places.

Reply:
What is the reason not to import opentracing?